README.md

```diff
@@ -72,7 +73,8 @@ Can be a [glob](https://github.com/isaacs/node-glob#glob-primer) or and `Array`
 
 | Property | Description                                                                                                 | Default                              |
 | -------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
-| `path`   | **Required.** A [glob](https://github.com/isaacs/node-glob#glob-primer) to identify the files to upload.    | -                                    |
+| `url`    | **path or url is required.** This provides the ability to add links to releases. E.g. URLs to container images.                           | -                                    |
```

Comment:
I would put the path up as this may still be the most commonly used feature; to weight it higher. @awcjack What do you think?